modules/002-deckhouse/webhooks/validating/d8-cluster-configuration

```diff
@@ -88,6 +88,17 @@ function __main__() {
       fi
     fi
 
+    # Not allow to downgrade more than 1 kube version
+    newKubernetesVersion=$(yq eval '.kubernetesVersion' - <<< "$newConfig")
+    # Possible kubernetes versions are "Automatic" or semver(1.28) versions
+    if [ "${newKubernetesVersion}" != "Automatic" ]; then
+      maxUsedControlPlaneKubernetesVersion=$(context::jq -r '.review.request.object.data."maxUsedControlPlaneKubernetesVersion"' | base64 -d)
+      if [ $((${maxUsedControlPlaneKubernetesVersion:2} - ${newKubernetesVersion:2})) -gt 1 ]; then
```

Comment:
It is necessary to take into account the change in the major version

Reply:
check minor and major version. I get it using cut command. After it check all same as in .go file